asn1-combinators.opam

```diff
@@ -13,7 +13,7 @@ build: [ ["dune" "subst"] {pinned}
 depends: [
   "ocaml" {>="4.05.0"}
   "dune" {>= "1.2.0"}
-  "cstruct" {>= "1.6.0"}
+  "cstruct" {>= "6.0.0"}
```

Comment:
what is the need for this change? the diff below works in earlier cstruct versions as well.

Reply:
Yes, I thought about that in the night.